README.md

```diff
@@ -177,7 +177,41 @@ toggle is enabled or not.
 unleash.IsEnabled("app.ToggleX")
 ```
 
-### 4. Stop unleash
+### 4. Experimental: Streaming Mode (Server-Sent Events)
```

Comment:
we made a decision to keep this out of the main readme in all the SDKs until we want people to start using it. I'd remove for now